README.org

```diff
@@ -242,10 +242,15 @@ The following snippet will refresh caches when one of the bib files change:
 
 (add-hook 'LaTeX-mode-hook (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
 (add-hook 'org-mode-hook (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
+
+(defun refresh-bib-file-watches () (interactive) (bibtex-actions-with-filenotify-refresh #'bibtex-actions-refresh))
```

Comment:
```suggestion
(add-hook 'LaTeX-mode-hook
          (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
(add-hook 'org-mode-hook
          (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))

(defun refresh-bib-file-watches ()
  (interactive) (bibtex-actions-with-filenotify-refresh #'bibtex-actions-refresh))
```

Just adjusted the formatting, but is the last one right; defining the function here?

Reply:
It refresh the watches but doesn't refresh the cache. Doing both seems like the more natural thing so I added that too.

Reply:
So far just reading the README: how do I include note and/or PDF paths?

Reply:
> So far just reading the README: how do I include note and/or PDF paths?

`bibtex-actions-with-filenotify-local` and `bibtex-actions-with-filenotify-global` both accept an optional argument which is a list of extra filepaths to place watches on. 

`bibtex-actions-with-filenotify-refresh` aceepts two extra optional arguments.